configure.ac

```diff
@@ -768,6 +771,34 @@ AS_IF([test x$with_lustre != xno],[
        MOTR_LINUX_SENDFILE
 ])
 
+#
+# Checking libfabric library----------------------------------------------- {{{1
```

Comment:
Have a look at `# Checking libyaml --------` section in the same file and consider implementing `libfabric` checks using similar structure. Also, you need to use `MOTR_SEARCH_LIBS()` to check that `-lfabric` is really available.